### PR TITLE
Group Labels into One Accessibilty Element on Explore Article Preview Cell

### PR DESCRIPTION
**Pull request by joewalsh**

### CODE REVIEW

**Comment by joewalsh** [comment]
adding WIP tag as subview order doesn't necessarily equal visual order. looking at other options

File: Wikipedia/Code/GroupedAccessibilityView.swift [hunk 1]
```diff
@@ -0,0 +1,71 @@
+import UIKit
+
+class GroupedAccessibilityView: UIView {
+
+    var arrangedAccessibilityViews: [UIView] = []
+    
+    //convienene outlets
+    @IBOutlet weak var accessibilityView0: UIView?
```
Comment (coreyfloyd) [inline Wikipedia/Code/GroupedAccessibilityView.swift]: Do we we use an outlet collection? or does that matter for this purpose?

**Comment by coreyfloyd** [comment]
👍

File: Wikipedia/Code/GroupedAccessibilityView.swift (hunk 1 above)
Comment (joewalsh) [inline Wikipedia/Code/GroupedAccessibilityView.swift]: Can't really guarantee the order of an outlet collection - we could tag all the views and sort on that but this seems easier? Open to suggestion

**Comment by coreyfloyd** [comment]
👍 waiting on travis

**Pull request merged**